handler.go

```diff
@@ -4,22 +4,38 @@ import (
 	"encoding/json"
 	"log"
 	"net/http"
+	"strconv"
 )
 
 func courseSimpleSearchHandler(w http.ResponseWriter, r *http.Request) {
-	q := r.URL.Query()
-	courseName := q.Get("course_name")
-	courseNameFilterType := q.Get("course_name_filter_type")
-	courseOverview := q.Get("course_overview")
-	courseOverviewFilterType := q.Get("course_overview_filter_type")
-	filterType := q.Get("filter_type")
-	limit := q.Get("limit")
-	offset := q.Get("offset")
 
-	// if !(filterType == "and" || filterType == "or") {
-	// 	w.WriteHeader(http.StatusBadRequest)
-	// 	return
-	// }
+	//Validate request
+	if r.Method != "POST" {
+		w.WriteHeader(http.StatusBadRequest)
```

Comment:
```suggestion
		w.WriteHeader(http.StatusMethodNotAllowed)
```

「405 Method Not Allowed」にあたりそう
- [405 Method Not Allowed - HTTP | MDN](https://developer.mozilla.org/ja/docs/Web/HTTP/Status/405)